src/text/index.tsx

```diff
@@ -29,24 +29,39 @@ interface TextProps {
   readonly tag?: TextTagType
 }
 
+const replaceNewLineWithBR = (str: string): React.ReactNode =>
+  str
+    .split('\n')
+    .map(line => <Fragment>{line}</Fragment>)
+    .reduce((acc, curr) => (
+      <Fragment>
+        {acc}
+        <br />
+        {curr}
+      </Fragment>
+    ))
+
 const Text = ({
   className,
   children,
   display = TextDisplayType.BODY,
   tag = TextTagType.SPAN,
-}:TextProps) => {
+}: TextProps) => {
   const baseClassName = 'kirk-text'
   const displayClassName = `${baseClassName}-${display}`
   const Tag = tag
 
+  let content = children
+  if (typeof children === 'string') {
+    content = replaceNewLineWithBR(children)
```

Comment:
Can you make it a prop `nl2br` (default to true)? Not sure about the naming though :)

Reply:
@wakooka  in what case do you think we'll need to disable `nl2br` ?